reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md

```diff
@@ -46,20 +50,18 @@ sudo apt-get install -y powershell
 pwsh
 ```
 
-After registering the Microsoft repository once as superuser, from then on, you just need to use `sudo apt-get upgrade powershell` to update it.
+As superuser, Register the Microsoft repository. From then on, you just need to use `sudo apt-get upgrade powershell` to update the installation.
```

Comment:
Seems odd for `Register` to be capitalized here.  Also, it seems you're cleaning up the markdown with semantic linefeeds (awesome!) but missed this one.

Reply:
oops. I will fix that.